xsamtools/misc_utils.py

```diff
@@ -0,0 +1,17 @@
+import subprocess
```

Comment:
This file should be renamed to either "xsamtools/utils.py", or something like "xsamtools/subproc.py".